piqe_ocp_lib/api/resources/ocp_nodes.py

```diff
@@ -343,6 +343,26 @@ def mark_node_unschedulable(self, node_name: str) -> Optional[ResourceInstance]:
                 logger.error("Exception encountered while marking node unschedulable: %s\n", e)
         return api_response
 
+    def make_node_uncordon(self, node_name: str) -> Optional[ResourceInstance]:
```

Comment:
Unless I am mistaken, I think making cordon is equivalent to making the node unschedulable so this should be the opposite, as in 'make_node_cordoned' correct?